schema/queue/configuration/QueueConfiguration.go

```diff
@@ -21,8 +21,10 @@ import (
 	"os"
 )
 
-type MqRouterFilterConfiguration struct{}
-
+type MqRouterFilterConfiguration struct {
+	Metadata interface{} `json:"metadata"`
+	Message  interface{} `json:"message"`
```

Comment:
Why this is just an interface? I think it should be the certain structure

Reply:
because it can be list or map type, not only one specific and for this situation I thought I should use interface